drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/pages/common/DecisionTableColumnViewUtils.java

```diff
@@ -37,6 +39,9 @@
 
 public class DecisionTableColumnViewUtils {
 
+    private static final List<Element> popoverElementRegistrations = new ArrayList<>();
+    private static final List<HTMLElement> popoverHTMLElementRegistrations = new ArrayList<>();
+
```

Comment:
Why do we need to add these two lists? 🤔

Reply:
They don't extend a common ancestor (other than ```Object```) :-(

Some Popovers are on older ```Widget.getElement()``` and some on Errai's ```HTMLElement```

Reply:
@manstis Hmm... but.. I think that these two lists are never used. Maybe I'm missing something.

Reply:
@karreiro Ahhhhhhhhhhhhh.. ok. I'll delete them.. (the ```Popover``` helper used to be in this class).

Reply:
:-)